guide/canvas/README.md

```diff
@@ -0,0 +1,334 @@
+## Setting up Canvas
+
+<p class="tip">For this guide, we will be using `canvas@next`. At the time of writing, that is `canvas@2.0.0-alpha.12`, so ensure your `package.json` has that or a similar version after installation.</p>
+
+Canvas is an image manipulation tool that allows you to basically modify pictures with code. We will explore how to use this module in a heavily requested feature: guild member welcome messages. But first, we must go through the intense labour of installing canvas. I highly reccommend using a linux distro because it is much easier to install.
+
+<p class="warning">This guide assumes you have read the [async await](/additional-info/async-await) portion of the guide.</p>
```

Comment:
I think rewording it to say "you should know async/await before continuing" instead of saying "we assume you've read our guide page on async/await" would be better, since most people learn about async/await externally. (Keep the guide link though; I like the self promotion. 😜)